icu4c/source/test/cintltst/cldrtest.c

```diff
@@ -1028,6 +1028,11 @@ static void VerifyTranslation(void) {
                     end = ures_getSize(resArray);
                 }
 
+                if ((uprv_strcmp(currLoc,"sd") == 0 || uprv_strncmp(currLoc,"sd_",3) == 0) && 
```

Comment:
Nit: Could this just be `(uprv_strncmp(currLoc,"sd", 2) == 0)` instead?

Reply:
Also thanks for adding this Peter!

Reply:
> Nit: Could this just be `(uprv_strncmp(currLoc,"sd", 2) == 0)` instead?

No because there might  be a locale with a 3-letter language code starting with sd

Reply:
Good point. Thanks Peter!